extensions/interactions/guppy.html

```diff
@@ -1,24 +1,48 @@
-<link rel="stylesheet" href="/third_party/static/guppy-b5055b/build/guppy-default.min.css">
+<link rel="stylesheet" href="/third_party/static/guppy-f944022/build/guppy-default.min.css">
+<link rel="stylesheet" href="/third_party/static/guppy-f944022/build/guppy-default-osk.min.css">
 <script>
   // Use the normal default browser eventlistener without passive set to false.
   // TODO(#8833): Use passive listeners when guppy supports it.
   EventTarget.prototype.addEventListener = (
     EventTarget.prototype.addEventListener._original);
 </script>
-<script type="text/javascript" src="/third_party/static/guppy-b5055b/build/guppy.min.js"></script>
+<script type="text/javascript" src="/third_party/static/guppy-f944022/build/guppy.min.js"></script>
+<script type="text/javascript" src="/third_party/static/guppy-f944022/build/guppy_osk.min.js"></script>
 
 <script type="text/javascript">
+  var osk = new GuppyOSK({'goto_tab':'arithmetic'});
+  Guppy.use_osk(osk);
   Guppy.init({
-    "path": "/third_party/static/guppy-b5055b/build"
+    'path': '/third_party/static/guppy-f944022/build'
+  });
+  Guppy.configure('buttons', ['osk', 'controls']);
+  Guppy.configure('empty_content', '\\color{grey}{\\text{\\small{Type a formula here.}}}')
+  document.addEventListener('keydown', function (event) {
+    if(event.key == 'Escape') {
+      osk.detach();
+    }
   });
 </script>
 
 <style>
+  .guppy_help {
+    top: 4% !important;
+    left: 15% !important;
+    width: 70%;
+    z-index: 1051;
+  }
+
+  .guppy_osk {
+    z-index: 1051;
+  }
+
   .guppy_active {
+    width: 550px;
     border: 1px solid rgba(81, 203, 238, 1);
   }
 
   .guppy_inactive {
+    width: 550px;
     background: #ffffff;
     border: 1px solid black;
```

Comment:
Can you change this to RGBA or use HEX values?

Reply:
Done.